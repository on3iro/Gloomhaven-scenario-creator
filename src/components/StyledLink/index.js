/**
 * A simple styled react-router Link element.
  */

import { Link } from 'react-router';
import styled from 'styled-components';


const StyledLink = styled(Link)`
  display: inline-block;
  margin: 5px;
  width: 100px;
  text-decoration: none;
  color: #fff;
  cursor: pointer;
  outline: 0;
`;

export default StyledLink;
