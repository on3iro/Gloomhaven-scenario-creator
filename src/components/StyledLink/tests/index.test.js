import React from 'react';
import { shallow } from 'enzyme';
import { Link } from 'react-router-dom';

import StyledLink from '../index';


describe('<StyledLink />', () => {
  it('should render <Link />', () => {
    const wrapper = shallow(<StyledLink to="/" />);
    expect(wrapper.find(Link).length).toMatchSnapshot();
  });
});
