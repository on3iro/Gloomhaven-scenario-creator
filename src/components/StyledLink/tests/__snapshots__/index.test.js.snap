exports[`<StyledLink /> should render <Link /> 1`] = `1`;
